jwst/cube_build/ifu_cube.py

```diff
@@ -1306,13 +1309,47 @@ def setup_ifucube_wcs(self):
             log.debug(f'Working on data from {this_a}, {this_b}')
             n = len(self.master_table.FileMap[self.instrument][this_a][this_b])
             log.debug('number of files %d', n)
+            
+            # find the median center declination if we have an offset file
+            if self.offsets is not None:
+                decs = []
+                for k in range(n):
+                    input_file = self.master_table.FileMap[self.instrument][this_a][this_b][k]
+                    input_model = datamodels.open(input_file)
```

Comment:
This datamodels.open call seems odd - is looks like that element of the FileMap ought to already be an open datamodel, not a filename?